article/app/views/fragments/mainMedia.scala.html

```diff
@@ -34,21 +34,7 @@
                 </figure>
             }
         } else {
-            @if(article.hasMainAudio) {
```

Comment:
we don't support this yet. If we do, they will need an appropriate media id too (probably jsouped, like videos).